Mediator-Error_Handling.md

```diff
@@ -66,8 +66,8 @@ This table defines the expected the behavior of the mediator in different scenar
 
 - **G8** - When a parsing error from the decrypted message.
   - G8A - ignore (no effect/output)
-  - G8B - (sync!) Send a `e.p.msg` If the plaintext message is malformed 
-  - G8C - Send a `e.p.msg.<PIURI>` If the plaintext message is malformed or if parsing into a specific protocol's data model fails.
+  - G8B - (sync!) Send an `e.p.msg` If the plaintext message is malformed 
+  - G8C - Send an `e.p.msg.<PIURI>` If the plaintext message is malformed or if parsing into a specific protocol's data model fails.
```

Comment:
- G8C - Send an `e.p.msg.<PIURI>` If the plaintext message is malformed or parsing into a specific protocol's data model fails.